osquery/tables/system/posix/last.cpp

```diff
@@ -6,18 +6,31 @@
  *  the LICENSE file found in the root directory of this source tree.
  */
 
-#include <vector>
-#include <string>
-
 #include <utmpx.h>
 
 #include <osquery/core.h>
 #include <osquery/tables.h>
-#include <osquery/utils/system/time.h>
 
 namespace osquery {
 namespace tables {
 
+namespace impl {
+
+void genLastAccessForRow(const utmpx* ut, QueryData& results) {
```

Comment:
const reference, not pointer
```suggestion
void genLastAccessForRow(const utmpx& ut, QueryData& results) {
```